questions.json

```diff
@@ -165,4 +165,85 @@
         "option4": "Hearing",
         "correct_answer": "option1"
     }]
+    
+    {
+        "question": "Which Year was the United Nations founded?",
+        "option1": "1945",
+        "option2": "1947",
+        "option3": "1949",
+        "option4": "1951",
+        "correct_answer": "option 1"
```

Comment:
I just noticed.. You have spaced in correct options. "option 1". but if you see previous questions. the answers are without spaces. 'option1' etc which should be the case.

Reply:
ok. Noted